src/components/DatePicker/RelativeDateRange.jsx

```diff
@@ -4,24 +4,24 @@ import React from "react";
 import { Button, Select } from "../..";
 
 const options = {
-    YESTERDAY: "P1D:last",
-    TODAY: "P1D:current",
+    YESTERDAY: "P1D,last",
```

Comment:
can you use the values instead of the keys from the below map 

for eg. LAST_WEEK you are using "P1W,last" instead you use "P1W,last week" 
            LAST_QUARTER you are using "P1W,last" instead you can use "P3M,first day of last quarter"
            THIS_MONTH =>  P1M,first day of this month
            
            
`  INTERVAL_START_MAP: {
        P1D: {
            last: 'yesterday',
            current: 'today',
            next: 'tomorrow',
            leading: 'today'
        },

        P1W: {
            last: 'last week',
            trailing: '-1 week',
            current: 'this week',
            leading: 'today',
            next: 'next week'
        },

        P1M: {
            last: 'first day of last month',
            trailing: '-1 month',
            current: 'first day of this month',
            leading: 'today',
            next: 'first day of next month'
        },

        // String "quarter" is not a standard PHP DateTime value when parsing a date,
        // but the API will understand it.
        P3M: {
            last: 'first day of last quarter',
            trailing: '-3 months',
            current: 'first day of this quarter',
            leading: 'today',
            next: 'first day of next quarter'
        },

        P1Y: {
            last: 'first day of January last year',
            trailing: '-1 year',
            current: 'first day of this year',
            leading: 'today',
            next: 'first day of January next year'
        }
    },`